resources/core/charts/namespace-controller/README.md

```diff
@@ -0,0 +1,7 @@
+# Namespace controller
+
+## Overview
+
+Namespace controller is the name of the controller that injects limit ranges and resource quotas into user-created Namespaces. It also adds two default roles, `kyma-reader-role` and `kyma-admin-role`, which it then copies into user-created Namespaces.
+
+Read the [namespace-controller](https://kyma-project.io/docs/root/kyma#details-environments) document to learn more about Namespaces in Kyma.
```

Comment:
I guess the link also must be updated.

Reply:
@tomekpapiernik will be updating docs in another PR. I cannot change this link until rest of docs is also renamed.